server.babel.js

```diff
@@ -1,8 +1,9 @@
 //  enable runtime transpilation to use ES6/7 in node
 
 var fs = require('fs');
+var path = require('path');
 
-var babelrc = fs.readFileSync('./.babelrc');
+var babelrc = fs.readFileSync(path.resolve(__dirname, 'node_modules/universal-redux/.babelrc'));
```

Comment:
Good question. The reason this file exists is so that this project can have ES6 in its server files and have transpilation on the fly. I think you're right that for the purpose of this project a `.babelrc` should live on project root, be configured in this file, and then be passed in through the universal-redux.config.js rather than reaching into `node_modules`, that way the internal concerns of universal-redux (such as react-transform-hmr configuration) don't leak into the project .babelrc configuration.